tests/Message/BatchMessageTest.php

```diff
@@ -109,15 +111,17 @@ public function testMaximumBatchSize()
         $this->assertEquals(1, count($message['to']));
     }
 
-    public function testAttributeResetOnEndBatchMessage()
+    public function testRecipientAttributeResetOnEndBatchMessage()
     {
         $this->batchMessage->addToRecipient('test-user@samples.mailgun.org', ['first' => 'Test', 'last' => 'User']);
         $this->batchMessage->setFromAddress('samples@mailgun.org', ['first' => 'Test', 'last' => 'User']);
         $this->batchMessage->setSubject('This is the subject of the message!');
         $this->batchMessage->setTextBody('This is the text body of the message!');
         $this->batchMessage->finalize();
         $message = NSA::getProperty($this->batchMessage, 'message');
-        $this->assertTrue(true, empty($message));
```

Comment:
This test is poorly written. I also think that this test is wrong. We should never reset the message. That would be catastrophic for the "auto send" feature

Reply:
Agree, this test seems wrong